pom.xml

```diff
@@ -235,19 +234,19 @@
                    <dumpDetails>true</dumpDetails>
                    <previousVersion>${previous.version}</previousVersion>
                    <excludes>
-                     <exclude>parquet/hadoop/thrift/**</exclude>
-                     <exclude>parquet/thrift/projection/**</exclude>
-                     <exclude>parquet/thrift/ThriftSchemaConverter</exclude>
-                     <exclude>parquet/filter2/**</exclude>
-                     <exclude>parquet/org/**</exclude>
-                     <exclude>parquet/column/**</exclude>
-                     <exclude>parquet/hadoop/ParquetInputSplit</exclude>
+                     <exclude>org/apache/parquet/hadoop/thrift/**</exclude>
+                     <exclude>org/apache/parquet/thrift/projection/**</exclude>
+                     <exclude>org/apache/parquet/thrift/ThriftSchemaConverter</exclude>
+                     <exclude>org/apache/parquet/filter2/**</exclude>
+                     <exclude>org/apache/parquet/column/**</exclude>
+                     <exclude>org/apache/parquet/hadoop/ParquetInputSplit</exclude>
+                     <exclude>shaded/**</exclude> <!-- shaded by parquet -->
+                     <exclude>parquet/**</exclude> <!-- shaded by parquet-format -->
```

Comment:
Yeah, we need to take a look at the packaging for parquet-tools. When it moved to a shaded jar that messed things up for me. We will package the old way, as an assembly tarball.